test/test_intensify/test_intensify.py

```diff
@@ -515,25 +550,18 @@ def target(x):
         # run incumbent
         config, _ = intensifier.get_next_challenger(challengers=None,  # since incumbent is run, no configs required
                                                     chooser=None)
-        self.assertEqual(config, self.config1)
+        self.assertEqual(config, self.config2)
         self.assertEqual(intensifier.stage, IntensifierStage.RUN_INCUMBENT)
         inc, _ = intensifier.eval_challenger(challenger=config, incumbent=inc, run_history=self.rh, )
 
-        self.assertEqual(inc, self.config1)
```

Comment:
Is there a reason why you removed these checks?

Reply:
We already test the same thing in the same test case, so I thought it was redundant.